app/src/main/java/com/example/playlistmarket/data/db/entity/TrackEntity.kt

```diff
@@ -0,0 +1,37 @@
+package com.example.playlistmarket.data.db.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import androidx.room.TypeConverter
+import androidx.room.TypeConverters
+import java.util.Date
+
+@Entity(tableName = "favorite_tracks")
+@TypeConverters(DateConverter::class)
+data class TrackEntity(
+    val previewUrl: String,
+    val trackName: String,
+    val artistName: String,
+    val trackTimeMillis: Long,
+    val artworkUrl100: String,
+    @PrimaryKey @ColumnInfo(name = "track_id")
+    val trackId: Int,
```

Comment:
```suggestion
    @PrimaryKey 
    @ColumnInfo(name = "track_id")
    val trackId: Int,
```
По традиции id всегда пишут первым параметром